src/tree.js

```diff
@@ -117,7 +135,11 @@ export let flattenTree = (next = traverse) => (buildPath = treePath()) =>
 
 export let flatLeaves = (next = traverse) => _.reject(next)
 
-export let tree = (next = traverse, buildIteratee = _.identity) => ({
+export let tree = (
+  next = traverse,
```

Comment:
Can we add a comment to change this on the next breaking change to mirror `writeNode`

Reply:
So I've been thinking about this - we could do this, but I'm not sure I understand a scenario where you need to know the current index or parents to understand how to traverse into children, which means this would never be used (but would be very complex to track passing all the other props everywhere). Especially with the latest change that makes it so you likely don't ever need to think about `writeNode` (since the default uses `next` and `index`), I think we can let this go.